src/saltext/proxmox/clouds/proxmox.py

```diff
@@ -460,7 +460,7 @@ def avail_locations(call=None):
     return ret
 
 
-def avail_images(call=None, location="local"):
+def avail_images(kwargs=None, call=None):
```

Comment:
**question (blocking):** will your change still work if the `call` parameter is first? we don't normally want to re-order parameters in case external usage isn't using keywords.

Reply:
Let's keep `call` as the first parameter for now. I'm probably being too safe, but I don't see any reason to change it and I'd rather err on the side of caution.